### PR TITLE
Reusable table view header pt 2

### PR DESCRIPTION
**Pull request by mitchdowney**

### CODE REVIEW

**Comment by mitchdowney** [comment]
@kreonjr ready 4 review

**Comment by mitchdowney** [comment]
@kreonjr whoops I didn't mean for the Status Message stuff to be in this PR. Sorry bout that.

For that commit, would you recommend a different name for the ErrorMessages enum? I'm not sure what to call that...

File: Podverse/EpisodesTableViewController.swift [hunk 1]
```diff
@@ -378,17 +382,17 @@ extension EpisodesTableViewController:FilterSelectionProtocol {
         
         let alert = UIAlertController(title: "Show", message: nil, preferredStyle: .alert)
```
Comment (kreonjr) [inline Podverse/EpisodesTableViewController.swift:383]: Since there are more than 3 actions, this should probably be an actionSheet

**Comment by kreonjr** [comment]
For the `ErrorMesages` question, it looks like there is no reason for those to be in an enum. they are not used as an options. Only the string is used. You could probably create a struct called `Strings` that contains a couple of other structs like for example `Errors` and `Actions`,  that only contain  a bunch of string properties each so when you need i.e noClipsAvailable, you can just say, `Strings.Errors.noClipsAvailble`. So in general if the enum is not used as an option, you don't really need it. A struct with constants is probably better.

**Comment by mitchdowney** [comment]
@kreonjr ready 4 review pt 2

**Comment by mitchdowney** [comment]
Whoops dang it, I pushed the changes you requested on this PR to the other PR I just made. Sorry bout that. I think this one can be merged anyway...

**Pull request merged**